TheraLang.BLL/Services/File/LocalFileService.cs

```diff
@@ -38,5 +36,32 @@ public async Task<Uri> SaveFile(Stream stream, string fileExtension)
 
             return new Uri($"/fileuploads/{filename}", UriKind.Relative);
         }
+
+        public Task RemoveFile(string fileUrl)
+        {
+            var fileName = Path.GetFileName(fileUrl);

```

Comment:
honestly it seems like this method does to many things, maybe it's worth to split is somehow SOLID?